bftengine/tests/keyManager/KeyManager_test.cpp

```diff
@@ -261,7 +261,7 @@ TEST(ReplicaKeyStore, rotate) {
 // }
 
 TEST(ClusterKeyStore, push) {
-  ReservedPagesMock rpm(7, true);
+  ReservedPagesMock rpm(8, true);
```

Comment:
In order not to depend on linking issues, since ResPageClient has static initialization, it's better to use ReservedPages::totalNumberOfPages() instead of hardcoded value